### PR TITLE
Readmes rach java1

### PR DESCRIPTION
**Pull request by Rachael-E**
First batch of my Java Android readme makeovers, @sorenoid could you take a first look please when you have a moment, thanks.

### CODE REVIEW

File: java/query-map-image-sublayer/README.md [hunk 1]
```diff
@@ -1,27 +1,37 @@
...
-1. After loading the layer, get its sublayers you want to query with `(ArcGISMapImageSublayer) layer.getSubLayers().get(index)`.
-1. Load the sublayer, and then get its `ServiceFeatureTable` with `sublayer.getTable()`.
-1. Create `QueryParameters`. You can use the where clause to query against a table attribute or set the parameters geometry to limit the results to an area of the map.
-1. Call `sublayerTable.queryFeaturesAsync(queryParameters)` to get a `FeatureQueryResult` with features matching the query.
-1. Iterate through the result features to use them however you wish. 
+
+1. Create an `ArcGISMapImageLayer` object using the URL of an image service.
+2. After loading the layer, get the sublayer you want to query with `(ArcGISMapImageSublayer) layer.getSubLayers().get(index)`.
+3. Load the sublayer, and then get its `ServiceFeatureTable` with `sublayer.getTable()`.
+4. Create `QueryParameters`. You can use `queryParameters.setWhereClause(sqlQueryString)` to query against a table attribute and/or set `queryParameters.setGeometry(extent)` to limit the results to an area of the map.
```
Comment (sorenoid) [inline java/query-map-image-sublayer/README.md]:
```suggestion
4. Create a `QueryParameters` object. You can use `queryParameters.setWhereClause(sqlQueryString)` to query against a table attribute and/or set `queryParameters.setGeometry(extent)` to limit the results to an area of the map.
```

Comment (sorenoid) [reply java/query-map-image-sublayer/README.md]: It looks like we use complete sentences for the "How it works" section, but I could be wrong!

**Comment by Rachael-E** [comment]
thanks @sorenoid, nice catch! how's it looking now, happy for this to go to second review?

File: java/query-map-image-sublayer/README.md (hunk 1 above)
Comment (sorenoid) [reply java/query-map-image-sublayer/README.md]: yes LGTM!

**Comment by Rachael-E** [comment]
@hud10837 please could you do a second review?

File: java/project/README.md [hunk 2]
```diff
@@ -1,20 +1,31 @@
...
-Being able to project between spatial references is fundamental to a GIS. An example of when you would need to re-project data is if you had data in two different spatial references, but wanted to perform an intersect analysis with the `GeometryEngine::intersect` function. This function takes two geometries as parameters, and both geometries must be in the same spatial reference. If they are not, you could first use `GeometryEngine::project` to convert the geometries so they match.
 
-![Project App](project.png)
+Project a point from one spatial reference to another.
+
+![Image of project](project.png)
+
+## Use case
+
+Being able to project between spatial references is fundamental to a GIS. An example of when you would need to re-project data is if you had data in two different spatial references, but wanted to perform an intersect analysis with the `GeometryEngine.intersect` function. This function takes two geometries as parameters, and both geometries must be in the same spatial reference. If they are not, you could first use `GeometryEngine.project` to convert the geometries so they match.
```
Comment (hud10837) [inline java/project/README.md]: Is it meant to be "a GIS"? I'm unfamiliar with how people talk about GIS 😅

File: java/raster-function-service/README.md [hunk 3]
```diff
@@ -1,24 +1,43 @@
...
+![Image of apply raster function to raster from service](raster-function-service.png)
+
+## Use case
+
+Raster functions allow processing operations that can be applied to one or more rasters on the fly. Functions can be applied to rasters that come from a service. A land survey agency may apply hillshade and aspect functions to rasters with elevation data in order to better determine the topography of a landscape and to make further planning decisions.
 
 ## How to use the sample
-Click on the button to apply the Raster service.
+
+The raster function is applied automatically when the sample starts and the result is displayed.
```
Comment (hud10837) [inline java/raster-function-service/README.md]: This sample has a button that applies the function

File: java/raster-function-service/README.md [hunk 4]
```diff
@@ -1,24 +1,43 @@
...
-* Create `RasterFunction` using a `JSON` string.
-* Get raster function's arguments with `RasterFunctionArguments`.
-* Set image service raster in the raster function arguments using `setRaster(rasterName, raster)`.
-* Create `Raster` using the `RasterFunction`.
-* Create `RasterLayer` using `Raster`.
-* Add `RasterLayer`  to the map operational layers with `map.getOperationalLayers()`.
+
+1. Create the `ImageServiceRaster` referring to the image server URL.
+2. Create the `RasterFunction` from a JSON string.
+3. Get the name of the raster argument to the function with `rasterFunction.getArguments().getRasterNames().get(0)`.
```
Comment (hud10837) [inline java/raster-function-service/README.md]: This is the same case as the kotlin one; the workflow instead is to get the arguments and then get the names from it.

File: java/raster-layer-geopackage/README.md [hunk 5]
```diff
@@ -1,23 +1,34 @@
...
+# Raster layer (GeoPackage)
 
-![Raster Layer Geopackage App](raster-layer-geopackage.png)
+Display a raster contained in a GeoPackage.
+
+![Image of Raster Layer GeoPackage](raster-layer-geopackage.png)
+
+## Use case
+
+GeoPackages are designed to simplify file management and transfer. They can store raster files (as well as other types of data). An end-user wishing to transfer rasters from ArcGIS Pro or between runtime apps, might need to import raster files from GeoPackages into their map to view and analyze the data.
```
Comment (hud10837) [inline java/raster-layer-geopackage/README.md]: Nitpick: removed a comma before "might"
```suggestion
GeoPackages are designed to simplify file management and transfer. They can store raster files (as well as other types of data). An end-user wishing to transfer rasters from ArcGIS Pro or between runtime apps might need to import raster files from GeoPackages into their map to view and analyze the data.
```

File: java/project/README.md (hunk 2 above)
Comment (Rachael-E) [reply java/project/README.md]: yup, that's ok to say: think of it as "is fundamental to a geographic information system".

**Comment by Rachael-E** [comment]
thanks @hud10837! Changes made, happy with these?

**Pull request merged**